src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -651,11 +636,8 @@ private void insert(final SinkRecord record) {
       }
       // only get offset token once when service context is initialized
       // ignore ingested files
-      // if ingestionDeliveryGuarantee is AT_LEAST_ONCE, ignore offsetPersistedInSnowflake
-      // else discard the record if the record offset is smaller or equal to server side offset
-      if ((ingestionDeliveryGuarantee
```

Comment:
snowpipe doesnt support at least once anymore. remove this check since it is always false and in an OR